rest-api/test/unit_test/api_test/physical_measurements_api_test.py

```diff
@@ -15,7 +15,7 @@ def setUp(self):
     self.participant_id = self.create_participant()
```

Comment:
Can you add the equivalent of test_auto_pair_updates_participant_and_summary for this test (physical measurements API) as well?